psi4/share/psi4/databases/O24by5.py

```diff
@@ -0,0 +1,2628 @@
+TAGL["%s-%s-dimer" % (dbse, "23-1.0")] = """Dimer from He* - He*"""
+TAGL["%s-%s-monoA-CP" % (dbse, "23-1.0")] = """Monomer A He* - He*"""
+TAGL["%s-%s-monoB-CP" % (dbse, "23-1.0")] = """Monomer B He* - He*"""
+TAGL["%s-%s" % (dbse, "23-1.2")] = """ He* He* R=1.2"""
+TAGL["%s-%s-dimer" % (dbse, "23-1.2")] = """Dimer from He* - He*"""
+TAGL["%s-%s-monoA-CP" % (dbse, "23-1.2")] = """Monomer A He* - He*"""
+TAGL["%s-%s-monoB-CP" % (dbse, "23-1.2")] = """Monomer B He* - He*"""
+TAGL["%s-%s" % (dbse, "23-1.5")] = """ He* He* R=1.5"""
+TAGL["%s-%s-dimer" % (dbse, "23-1.5")] = """Dimer from He* - He*"""
+TAGL["%s-%s-monoA-CP" % (dbse, "23-1.5")] = """Monomer A He* - He*"""
+TAGL["%s-%s-monoB-CP" % (dbse, "23-1.5")] = """Monomer B He* - He*"""
+TAGL["%s-%s" % (dbse, "23-2.0")] = """ He* He* R=2.0"""
+TAGL["%s-%s-dimer" % (dbse, "23-2.0")] = """Dimer from He* - He*"""
+TAGL["%s-%s-monoA-CP" % (dbse, "23-2.0")] = """Monomer A He* - He*"""
+TAGL["%s-%s-monoB-CP" % (dbse, "23-2.0")] = """Monomer B He* - He*"""
+TAGL["%s-%s" % (dbse, "24-0.9")] = """ HF CO+ R=0.9"""
+TAGL["%s-%s-dimer" % (dbse, "24-0.9")] = """Dimer from HF - CO+"""
+TAGL["%s-%s-monoA-CP" % (dbse, "24-0.9")] = """Monomer A HF - CO+"""
+TAGL["%s-%s-monoB-CP" % (dbse, "24-0.9")] = """Monomer B HF - CO+"""
+TAGL["%s-%s" % (dbse, "24-1.0")] = """ HF CO+ R=1.0"""
+TAGL["%s-%s-dimer" % (dbse, "24-1.0")] = """Dimer from HF - CO+"""
+TAGL["%s-%s-monoA-CP" % (dbse, "24-1.0")] = """Monomer A HF - CO+"""
+TAGL["%s-%s-monoB-CP" % (dbse, "24-1.0")] = """Monomer B HF - CO+"""
+TAGL["%s-%s" % (dbse, "24-1.2")] = """ HF CO+ R=1.2"""
+TAGL["%s-%s-dimer" % (dbse, "24-1.2")] = """Dimer from HF - CO+"""
+TAGL["%s-%s-monoA-CP" % (dbse, "24-1.2")] = """Monomer A HF - CO+"""
+TAGL["%s-%s-monoB-CP" % (dbse, "24-1.2")] = """Monomer B HF - CO+"""
+TAGL["%s-%s" % (dbse, "24-1.5")] = """ HF CO+ R=1.5"""
+TAGL["%s-%s-dimer" % (dbse, "24-1.5")] = """Dimer from HF - CO+"""
+TAGL["%s-%s-monoA-CP" % (dbse, "24-1.5")] = """Monomer A HF - CO+"""
+TAGL["%s-%s-monoB-CP" % (dbse, "24-1.5")] = """Monomer B HF - CO+"""
+TAGL["%s-%s" % (dbse, "24-2.0")] = """ HF CO+ R=2.0"""
+TAGL["%s-%s-dimer" % (dbse, "24-2.0")] = """Dimer from HF - CO+"""
+TAGL["%s-%s-monoA-CP" % (dbse, "24-2.0")] = """Monomer A HF - CO+"""
+TAGL["%s-%s-monoB-CP" % (dbse, "24-2.0")] = """Monomer B HF - CO+"""
+
+# <<< Geometry Specification Strins >>>
+GEOS = {}
+
+GEOS["%s-%s-dimer" % (dbse, "1-0.9")] = qcdb.Molecule("""
+0   2
+C        -0.664078525     0.000000000     1.259898914
+N         0.292558536     0.000000000     1.928630081
+--
+0   1
+He        0.149064407     0.000000000    -1.619916319
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
+""")
+
+GEOS["%s-%s-dimer" % (dbse, "1-1.0")] = qcdb.Molecule("""
+0   2
+C        -0.680641237     0.000000000     1.439889616
+N         0.275995824     0.000000000     2.108620783
+--
+0   1
+He        0.165627119     0.000000000    -1.799907021
+
+units angstrom
+no_reorient
+no_com
+symmetry c1
```

Comment:
yeah, I'd drop the `no_reorient`, `no_com`, `symmetry c1` if they weren't added with deliberation. Spelling out the 0.9 (R_eq vs. Ang) in the docstring as Peter suggests would also be nice. otherwise, good to go, imo.

Reply:
Since these files were originally used for some of the results in the paper, I wanted them to reflect reference calculations with other software (including symmetry used).
Purged, everything except for `angstrom` as it should work without them and might save some resource, where spacial symmetry is applicable.